tests/end2end/features/caret.feature

```diff
@@ -320,3 +320,28 @@ Feature: Caret mode
         And the following tabs should be open:
             - data/caret.html
             - data/hello.txt (active)
+
+    # Search + caret mode
+
+    # https://bugreports.qt.io/browse/QTBUG-60673
+    @qt!=5.8.0 @qt!=5.9.0 @qt!=5.9.1 @qt!=5.9.2 @qt!=5.9.3 @qt!=5.9.4
```

Comment:
I'd definitely prefer having a dedicated marker for this - see 53e7d13c2d35c083bf820fbfbcd8271f8001f52e for an example. I think the best name would probably be `@qtbug60673` as that's surely unique.